packages/deployer/src/server/handlers/routes/observability/handlers.ts

```diff
@@ -0,0 +1,74 @@
+import type { Mastra, StorageGetAiTracesPaginatedArg } from '@mastra/core';
```

Comment:
why is the `StorageGetAiTracesPaginatedArg` in `@mastra/core` and not some deeper `@mastra/core/storage` or something?